storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst

```diff
@@ -0,0 +1,39 @@
+================================================================================
+|Percona XtraBackup| |release|
+================================================================================
+
+Percona is glad to announce the release of |Percona XtraBackup| |release| on
+|date|. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.13/>`_ and
+from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+All Percona software is open-source and free.
+
+Improvements and New Features
+================================================================================
+
+- :jirabug:`1548`: |Percona Xtrabackup| enables updating the
+  ib_buffer_pool file with the latest pages present in the buffer pool by
+  setting the ``innodb_buffer_pool_dump_now`` variable to `ON` .
+- :jirabug:`1609`: The ``--version_check`` option can be disabled at build time
+
+Bus Fixed
+================================================================================
+
+- :jirabug:`1536`: |Percona Xtrabackup| 2.4.11, MySQL 5.7.21-21-log Percona Server partial backup and errors in MySQL log
```

Comment:
xtrabackup did not delete missing tables from partial backup which led to error messages logged by server on startup